scripts/components/OpenSearch/install.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
```

Comment:
I would really prefer that this is python classes rather than shell scripts, its more testable and supports reuse